### PR TITLE
Search for music v1

### PR DESCRIPTION
**Pull request by reidbaker**

### CODE REVIEW

**Comment by reidbaker** [comment]
fixes issue #285

File: SoundStream/res/xml/searchable.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<searchable xmlns:android="http://schemas.android.com/apk/res/android"
+    android:label="@string/app_name"
+    android:hint="@string/search_hint" >
+</searchable>
```
Comment (ejohnson44) [inline SoundStream/res/xml/searchable.xml]: New Line?

File: SoundStream/src/com/thelastcrusade/soundstream/components/MusicLibraryFragment.java [hunk 2]
```diff
@@ -60,6 +62,8 @@
     private ServiceLocator<MusicLibraryService> musicLibraryServiceLocator;
     
     private MusicAdapter mMusicAdapter;
+    
+    private volatile String mQuery;
```
Comment (ejohnson44) [inline SoundStream/src/com/thelastcrusade/soundstream/components/MusicLibraryFragment.java:66]: what does setting this to volatile do? (aka why are we doing this)

**Comment by ejohnson44** [comment]
Overall everything looks pretty good to me considering that I don't know all that much about search things. The only thing I might say is that it would be nice if you added in some comments since I don't think many of us are familiar with the search activity concept (but I could be wrong)

**Comment by dgreenhalgh** [comment]
Yep, I'm in the same boat as @ejohnson44 here. Everything looks good, but it may help to have some context at more of a macro level

**Comment by dgreenhalgh** [comment]
@reidbaker walked us through this code at the Nov. 17 meeting. After the newline is fixed, I say :+1:

**Pull request merged**